### ISSUE TITLE
missing modules when I try to serve demo app locally

### ISSUE THREAD

**Issue by willhs** [angular-slider/ng5-slider#103]
I tried serving the demo app with `ng serve` after building with `npm run prepare:demo-app` as specified in the developer doc but I get:
```ERROR in src/demo-app/app/docs/docs.module.ts(4,53): error TS2307: Cannot find module './classes_change_context_.changecontext.component'.
src/demo-app/app/docs/docs.module.ts(5,41): error TS2307: Cannot find module './classes_options_.options.component'.
src/demo-app/app/docs/docs.module.ts(6,54): error TS2307: Cannot find module './classes_slider_module_.ng5slidermodule.component'.
src/demo-app/app/docs/docs.module.ts(7,43): error TS2307: Cannot find module './enums_options_.labeltype.component'.
src/demo-app/app/docs/docs.module.ts(8,49): error TS2307: Cannot find module './enums_pointer_type_.pointertype.component'.
src/demo-app/app/docs/docs.module.ts(11,54): error TS2307: Cannot find module './interfaces_options_.customstepdefinition.component'.
src/demo-app/app/docs/docs.module.ts(12,40): error TS2307: Cannot find module './modules_change_context_.component'.
src/demo-app/app/docs/docs.module.ts(13,34): error TS2307: Cannot find module './modules_options_.component'.
src/demo-app/app/docs/docs.module.ts(14,38): error TS2307: Cannot find module './modules_pointer_type_.component'.
src/demo-app/app/docs/docs.module.ts(15,39): error TS2307: Cannot find module './modules_slider_module_.component'.

webpack: Failed to compile.
```

I'm building on a windows machine.

**Comment by piotrdz** [comment]
This is coming from missing auto-generated doc files. This should be covered by `scripts/generate-demo-app-docs.js` which is run as part of `npm run prepare:demo-app`.

I'm not sure why it's not working for you, but, well, I haven't tried running it on Windows.

Do you get any errors in output from `npm run prepare:demo-app`? Do you see any files generated files under `src/demo-app/app/docs/`?

**Comment by willhs** [comment]
Looks like the backslashes in `src/demo-app/docs/docs.module` seem to be interpreted as an escape character:  
```import { ChangecontextchangecontextComponent } from './classes\_change_context_.changecontext.component';
import { OptionsoptionsComponent } from './classes\_options_.options.component';
```

**Comment by piotrdz** [comment]
D'oh! Of course, it is the path joining using backslashes on Windows. But this is a Typescript import which expects forward slashes.

I just fixed it on master and it will be available in next release.

**Comment by piotrdz** [comment]
Fix released with v1.2.0.
